internal/cmd/skupper/system/system.go

```diff
@@ -0,0 +1,122 @@
+package system
+
+import (
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/internal/cmd/skupper/common"
+	"github.com/skupperproject/skupper/internal/cmd/skupper/system/kube"
+	"github.com/skupperproject/skupper/internal/cmd/skupper/system/nonkube"
+	"github.com/skupperproject/skupper/pkg/config"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmdSystem() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "system",
+		Short: "non-kubernetes sites are static and Custom Resources need to be provided.",
+		Long: `Non-kubernetes sites can be created using the standard V2 site declaration 
```

Comment:
fixed (d666fa8e66de3ca56e63f113cf059418b3cbdd27)

Instead of adding it as the long description for "system" I used it as the long description for "system setup", given that is explaining the functionality of that specific command.